domain/ide/workspace/folder.go

```diff
@@ -553,53 +566,87 @@ func getIssuePerFileFromFlatList(issueList []snyk.Issue) snyk.IssuesByFile {
 }
 
 func (f *Folder) filterDiagnostics(issues snyk.IssuesByFile) snyk.IssuesByFile {
-	supportedIssueTypes := config.CurrentConfig().DisplayableIssueTypes()
+	supportedIssueTypes := f.c.DisplayableIssueTypes()
 	filteredIssuesByFile := f.FilterIssues(issues, supportedIssueTypes)
 	return filteredIssuesByFile
 }
 
+func (f *Folder) GetDeltaForAllProducts(supportedIssueTypes map[product.FilterableIssueType]bool) []snyk.Issue {
+	var deltaList []snyk.Issue
+	for filterableIssueType, enabled := range supportedIssueTypes {
+		if !enabled {
+			continue
+		}
+		p := filterableIssueType.ToProduct()
+		deltaIssueByFile, err := f.GetDelta(p)
+		if err == nil {
+			deltaList = append(deltaList, getFlatIssueList(deltaIssueByFile)...)
+		}
+	}
+	return deltaList
+}
+
 func (f *Folder) FilterIssues(issues snyk.IssuesByFile, supportedIssueTypes map[product.FilterableIssueType]bool) snyk.IssuesByFile {
 	logger := f.c.Logger().With().Str("method", "FilterIssues").Logger()
-
 	filteredIssues := snyk.IssuesByFile{}
+
+	if f.c.IsDeltaFindingsEnabled() {
+		issues = getIssuePerFileFromFlatList(f.GetDeltaForAllProducts(supportedIssueTypes))
+	}
+
 	for path, issueSlice := range issues {
 		if !f.Contains(path) {
 			logger.Error().Msg("issue found in cache that does not pertain to folder")
 			continue
 		}
 		for _, issue := range issueSlice {
 			// Logging here will spam the logs
-			if isVisibleSeverity(issue) && supportedIssueTypes[issue.GetFilterableIssueType()] {
+			if isVisibleSeverity(f.c, issue) && supportedIssueTypes[issue.GetFilterableIssueType()] {
```

Comment:
@andygongea should the summary depend on the visibility selection and activated/deactivated state of issue types?